packages/ui-components/src/components/calendar-range-dates-selector/calendar-range-dates-selector.tsx

```diff
@@ -50,21 +50,21 @@ export class KvCalendarRangeDatesSelector implements ICalendarRangeDatesSelector
 	};
 
 	private onClickDate = ({ detail: { event, payload: date } }: CustomEvent<IClickDateEvent>): void => {
-		const clickedDateMoment = fromISOToMoment(date);
+		const clickedDate = fromISO(date).startOf('day');
 
 		const [selectedStartDate, selectedEndDate] = this.getSelectedRangeDates();
 
 		// check if start date is not selected
 		if (selectedStartDate === undefined) {
-			this.selectRangeDates.emit({ event, payload: [formatDatetime(clickedDateMoment.startOf('day'))] });
+			this.selectRangeDates.emit({ event, payload: [formatDateTime(clickedDate.startOf('day'))] });
 			return;
 		}
 
 		// check if clicked date is the same as the selected date
-		if (isDateSame(clickedDateMoment, selectedStartDate)) {
+		if (isDateSame(clickedDate, selectedStartDate)) {
 			// check if end date is not selected
 			if (selectedEndDate === undefined) {
```

Comment:
could this comparison be simplified to `!selectedEndDate`? In his currently open PR @laraujo7 is using lodash's `isNil` method to verify falsy values and asked him if it was regarding linting issues, I'm waiting for the answer but I'm asking this because it's mostly to keep consistency in our boolean assertions

Reply:
Well, I think it can, indeed. I could use `isNil` as well if it is preferred. Maybe @djcouto should just comment if he has anything against, as I am not fully aware of everything that it can imply.

Reply:
I'm not a fan of overusing the lodash utilities. This is just an undefined check, don't see a reason for the function call to make the exact same check 🤷‍♂️

Reply:
all good for me :)

Reply:
But should I do `!selectedEndDate` instead?

Reply:
It doesn't bother me, go ahead 👍